python/tests/integration/toolbox/test_library_tool.py

```diff
@@ -137,3 +138,83 @@ def test_count_keys(object_and_mem_and_lmdb_version_store):
     assert lib_tool.count_keys(KeyType.SNAPSHOT_REF) == 1
     assert lib_tool.count_keys(KeyType.MULTI_KEY) == 1
     assert lib_tool.count_keys(KeyType.SNAPSHOT) == 0
+
+
+@pytest.mark.parametrize("use_time_index", [True, False])
+def test_read_data_key_from_version_ref(object_and_mem_and_lmdb_version_store, use_time_index):
```

Comment:
Why `object_and_mem_and_lmdb_version_store`? Just `mem` is probably fine

Reply:
Ah was just copying what the other fixtures did. Agreed that these are specifically testing the library tool functionality so mem is just fine.